xscen/io.py

```diff
@@ -235,280 +238,7 @@ def subset_maxsize(
         )
 
 
-def save_to_netcdf(
```

Comment:
Why move this away?

Reply:
I was trying to simplify things a bit. Will revert.